Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0-apache
+
+RUN apt-get update \
```

Comment:
I wonder if we could split the command for make it more simple and add more things to docker cache even if it add more layers